runtime/js/40_tty.js

```diff
@@ -16,7 +16,7 @@ function isatty(rid) {
   internals.warnOnDeprecatedApi(
     "Deno.isatty()",
     new Error().stack,
-    "Use `Deno.stdin.isTerminal()`, `Deno.stdout.isTerminal()` or `Deno.stderr.isTerminal()` instead.",
+    "Use `Deno.FsFile.isTerminal()`, `Deno.stdin.isTerminal()`, `Deno.stdout.isTerminal()` or `Deno.stderr.isTerminal()` instead.",
```

Comment:
Can you move `FsFile` to be the last - it's gonna be used rarely compared to the other ones